internal/provisioner/model/model.go

```diff
@@ -245,6 +245,12 @@ type ContourSpec struct {
 	// If the value is 0, the overload manager is disabled.
 	// defaults to 0.
 	EnvoyMaxHeapSizeBytes uint64
+
+	// WatchNamespaces specifies a subset of namespaces that the contour instance
+	// will be watching.
+	// Expect value should be comma separate namespaces
+	// For example, ns1,ns2,ns3
+	WatchNamespaces *string
```

Comment:
Probably makes sense for this to be a string slice too, and just convert to a comma-separated string for the contour deployment.